app/src/pages/patientResultPage/PatientResultPage.tsx

```diff
@@ -44,11 +43,11 @@ function PatientResultPage() {
     };
     const showWarning = patientDetails?.superseded || patientDetails?.restricted;
     const isGp = userIsGPAdmin || userIsGPClinical;
-    const pageHeader = 'Verify patient details';
+    const pageHeader = 'Patient details';
     useTitle({ pageTitle: pageHeader });
     return (
         <div style={{ maxWidth: 730 }}>
-            <BackButton />
+            <BackButton toLocation={routes.SEARCH_PATIENT} />
```

Comment:
added to prevent a loop stopping the user reaching the search patient page, cheers @mark-start-nhs